akka-docs/src/main/paradox/typed/failure-detector.md

```diff
@@ -0,0 +1,70 @@
+# Phi Accrual Failure Detector
+
+Remote Death Watch uses heartbeat messages and the failure detector to 
```

Comment:
Is it a name or is it `Remote death watch`  or is deathwatch the name perhaps `Remote DeathWatch`?

Reply:
The docs in many files use `death watch` but I think it's `* DeathWatch`. Updating.

Reply:
Well let's see, statistically speaking:

`death watch` = 38 matches over 21 files
`deathwatch` = 100 matches in 15 files

`Remote DeathWatch` for the win 🥇